examples/variational_autoencoder.py

```diff
@@ -39,7 +41,7 @@ def sampling(args):
 
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
+    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
@yaringal , true that it's not a big change. The main focus of this PR is on the change in the shape inference part and it's not a big deal to keep the same reference. If a paper is of particular interest it could maybe be integrated later. I'll let you guys decide!